content/about/use-case/_index.fr.md

```diff
@@ -5,24 +5,104 @@ description: Applications et axes directeurs
 weight: 60
 ---
 
-Le processus de planification de la circulation d’un train demande **10 ans d’anticipation** et implique la collaboration de différents services et acteurs pour transformer les besoins de mobilité des clients : **autorités organisatrices (AO)** et **entreprises ferroviaires (EF)** en **service annuel (SA)**.
+![OSRD presentation](osrd_presentation.png)
 
-Le projet OSRD à l’ambition de répondre aux besoins de simulation qui couvrent la totalité du processus, de la conception du plan de transport jusqu’à sa mise en exploitation.
+Le projet a pour ambition de répondre à une large gamme de besoins autour de la plannification ferroviaire :
+ - [conception des plans de transport](#conception-des-plans-de-transport)
+ - [études d'exploitation](#%C3%A9tudes-dexploitation)
+ - [gestion d'infrastructure](#gestion-dinfrastructure)
+ - [production de sillons de dernière minute](#sillons-de-derni%C3%A8re-minute)
 
-![OSRD presentation](osrd_presentation.png)
 
-Notre mission est de fournir un outil de simulation qui peut être utilisé dans **toutes les étapes de la phase de conception** (planification des horaires et études d’exploitation), sur une **échelle géographique nationale** (comprenant le réseau français) et avec une **granulométrie variable des détails**. De plus, dans la phase opérationnelle, le projet OSRD propose un service pour tracer des **sillons de dernières minutes** (tranche d’espace-temps durant laquelle l’infrastructure est affectée à la circulation d’un train spécifique) dans la capacité résiduelle théorique ou réelle du réseau ferroviaire, une fois les autres circulations ont été déjà planifiées.
+![produits OSRD](osrd_product.fr.png)
+
+
+## Conception des plans de transport
+
+La conception de plans de transport consiste à formuler une réponse[^plan-exploitation] aux
+schémas de dessertes[^schema-desserte]. OSRD fournit déjà un certain nombre de fonctionnés utiles à cette fin :
+ - base d'infrastructure ferroviaires à l'échelle microscopique
+ - base de donnée de matériel roulant
+ - possibilité de modifier l'infrastructure
+ - visualisation de grilles horaires
+ - application de différents types de marges sur les sillons, y compris une
+   marge économique, qui minimise l'énergie consommée
+
+Les perspectives futures comprennent notament :
+ - la détection automatique de conflits[^conflit]
+ - synthèse d'infrastructure macroscopique à partir d'infrastructure microscopique
```

Comment:
The adjectives "macroscopique" and "microscopique", should possibly be explained with more details.